source/dm_recipes.rst

```diff
@@ -0,0 +1,56 @@
+=======================
+Data Management Recipes
+=======================
+
+----------------
+Register context
+----------------
+
+Data Provider registers dataset
+===============================
+
+In order to make a dataset findable, a dataset must be registered in S-ENDA Find with appropriate metadata, which is indexed and exposed in CSW. The procedure is as follows:
+
+#. Create MMD XML record of metadata
+#. Translate from MMD to ISO19115
+#. Index in CSW
+#. Expose ISO19115 metadata via CSW to the internet
+
+This is done with the s-enda-find Virtual Machine (VM), for which a Vagrant configuration is available in https://github.com/metno/S-ENDA-Prototype.
+
+**TODO:** We need to describe how to do this:
```

Comment:
Do you mean here, how to create metadata records?
In this prototype, that not possible to do in a nice way, since both the Dynamic geo assets API and dashboard are mocks.

But perhaps you mean to document how to add static mmd records?

Reply:
ok - yes, then maybe as you say. We should also say what is mocked, and what is not. Could we add screenshots to demonstrate our thinking?